### PR TITLE
feat(thorvg): update thorvg version to 0.13.5

### PR DESCRIPTION
**Pull request by lhdjply**
### Description of the feature or fix

feat(thorvg): update thorvg version to 0.13.5

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) version [v3.4.12](https://github.com/szepeviktor/astyle/releases/tag/v3.4.12) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

File: src/libs/thorvg/tvgSwRenderer.cpp [hunk 1]
```diff
@@ -437,21 +437,13 @@ bool SwRenderer::target(pixel_t* data, uint32_t stride, uint32_t w, uint32_t h,
...
 
 
 bool SwRenderer::preRender()
 {
-#if LV_USE_DRAW_VG_LITE && LV_USE_VG_LITE_THORVG
-	return true;
-#else
     return rasterClear(surface, 0, 0, surface->w, surface->h);
-#endif
 }
```
Comment (kisvegabor) [inline src/libs/thorvg/tvgSwRenderer.cpp]: These should be kept.

**Review by kisvegabor — COMMENTED** [review]
Thank you!

File: src/libs/thorvg/thorvg_capi.h [hunk 2]
```diff
@@ -433,7 +435,7 @@ typedef enum {
 *
 * \return A new Tvg_Canvas object.
 */
-TVG_API Tvg_Canvas* tvg_swcanvas_create(void);
+TVG_API Tvg_Canvas* tvg_swcanvas_create();
```
Comment (kisvegabor) [inline src/libs/thorvg/thorvg_capi.h]: Please keep the `void` parameters in this file. I'm sending a PR to ThorVG about it.

Comment (XuNeo) [reply src/libs/thorvg/thorvg_capi.h]: Already done in https://github.com/thorvg/thorvg/pull/2316

Comment (kisvegabor) [reply src/libs/thorvg/thorvg_capi.h]: Great, I've closed mine: https://github.com/thorvg/thorvg/pull/2323

File: src/libs/thorvg/tvgSwRenderer.cpp (hunk 1 above)
Comment (lhdjply) [reply src/libs/thorvg/tvgSwRenderer.cpp]: done

File: src/libs/thorvg/thorvg_capi.h (hunk 2 above)
Comment (lhdjply) [reply src/libs/thorvg/thorvg_capi.h]: done

**Review by kisvegabor — APPROVED** [review]
Thank you!

**Review by FASTSHIFT — APPROVED** [review]

**Pull request merged**